src/plugins/vis_type_timeline/server/series_functions/graphite.js

```diff
@@ -82,7 +169,7 @@ export default new Datasource('graphite', {
       '&target=' +
       config.metric;
 
-    return fetch(URL)
+    return fetch(GRAPHITE_URL, { redirect: 'error' })
```

Comment:
Non-blocking: This might be a regression if customer is using allow list. 

If they trust an redirection url and put it in allow list, this block them from using that url.

Reply:
Since customer can enable both allowlist and blocklist, I will just keep the redirect.